common/source/docs/common-remoteid.rst

```diff
@@ -35,6 +39,8 @@ OpenDroneID Compatible devices (support included in ArduPlane 4.0 and later)
 - `Cube ID_CAN <https://docs.cubepilot.org/user-guides/cube-id/cube-id>`__
 - `Wurzbach Electronics <https://wurzbachelectronics.com/esp32-remote-id-development-set>`__
 - `mRo RemoteID <https://store.mrobotics.io/product-p/m10049.htm>`__
+- `Dronetag DRI <https://dronetag.cz/products/dri/>`__
```

Comment:
we should organize these alphabetically...not your problem...not a blocker and I can fix later if you wish